code/tests/_game_test_puppeteer.dm

```diff
@@ -56,6 +63,23 @@
 			var/mob/new_mob = origin_test.allocate(mob_type, T)
 			return new_mob
 
+/datum/test_puppeteer/proc/change_turf_nearby(turf_type, direction)
+	var/turf/T
+	if(direction)
```

Comment:
Direction == 0 is still a direction, isn't it? I think you might need a sentinel here, like -1.